File: openapi/code_samples/PHP/utilitypayments/get_transaction_by_id/code.php
<?php

$curl = curl_init();

curl_setopt_array($curl, array(
  CURLOPT_URL => 'https://utilities.reloadly.com/transactions/36',
  CURLOPT_RETURNTRANSFER => true,
  CURLOPT_ENCODING => '',
  CURLOPT_MAXREDIRS => 10,
  CURLOPT_TIMEOUT => 0,
  CURLOPT_FOLLOWLOCATION => true,
  CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
  CURLOPT_CUSTOMREQUEST => 'GET',
  CURLOPT_HTTPHEADER => array(
    'Accept: application/com.reloadly.utilities-v1+json',
    'Authorization: Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
  ),
));

$response = curl_exec($curl);

curl_close($curl);
echo $response;

File: openapi/code_samples/python/utilitypayments/get_transaction_by_id/code.py
import requests

url = "https://utilities.reloadly.com/transactions/36"

payload = ""
headers = {
  'Accept': 'application/com.reloadly.utilities-v1+json',
  'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.text)